english/33hours/wakingup.md

```diff
@@ -1,10 +1,13 @@
-The sun on your face is the first thing you become aware of and you instinctively put your hand up to shield your eyes
-from the brightness. You're still in your clothes and are lying in the middle of a bed in what looks to be some sort of
-cheap motel room. You have no idea where you are or how you got here.
+The sun on your face is the first thing you become aware of
+and you instinctively put your hand up to shield your eyes
+from the brightness.
+You're still in your
+clothes.And you are lying in the middle of a
+bed in what looks to be some sort [of](../Harold/Harold.md) cheap motel room.
+You have no idea where you are or how you got here.
 
-You sit up and immediately feel pain on the back of your head. Some dried blood cakes the pillow you were lying on. Your
-wallet and mobile phone are gone. In fact, your pockets are completely empty.
+You sit up and immediately feel pain on the back of your head. Some dried blood cakes the pillow you were lying on. Your wallet and mobile phone are gone. In fact, your pockets are completely empty.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.